Makefile

```diff
@@ -6,7 +6,7 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --tty --interactive --rm $(IMAGE_NAME) bundle exec rake
+	docker run --tty --interactive --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
```

Comment:
Ah, I see why this is needed now. My suggestion to use `--workdir` still stands.